modules/kwin.nix

```diff
@@ -564,6 +571,175 @@ in
+                type = with lib.types; nullOr (ints.between 0 999);
+                default = null;
+                example = 8;
+                description = "Right screen gap for krohnkite.";
+              };
+              bottom = lib.mkOption {
+                type = with lib.types; nullOr (ints.between 0 999);
+                default = null;
+                example = 8;
+                description = "Bottom screen gap for krohnkite.";
+              };
+              tiles = lib.mkOption {
+                type = with lib.types; nullOr (ints.between 0 999);
+                default = null;
+                example = 8;
+                description = "Gap between tiles for krohnkite.";
+              };
+            };
+
+            tileWidthLimit = {
+              enable = lib.mkOption {
+                type = with lib.types; nullOr bool;
+                default = null;
+                example = true;
+                description = "Whether to limit tile width for krohnkite.";
+              };
+              ratio = lib.mkOption {
+                type = with lib.types; nullOr (ints.between 1 100);
+                default = null;
+                example = 1.6;
+                description = "Tile width limiting ratio for krohnkite.";
+              };
+            };
+          };
+
+          layouts = {
+            btree.enable = lib.mkOption {
+              type = with lib.types; nullOr bool;
+              default = null;
+              example = true;
+              description = "Whether to enable binary tree layout for krohnkite.";
+            };
+            columns = {
+              enable = lib.mkOption {
+                type = with lib.types; nullOr bool;
+                default = null;
+                example = true;
+                description = "Whether to enable columns layout for krohnkite.";
+              };
+              balanced = lib.mkOption {
+                type = with lib.types; nullOr bool;
+                default = null;
+                example = true;
+                description = "Whether to enable balanced mode for columns layout.";
+              };
+            };
+            floating.enable = lib.mkOption {
+              type = with lib.types; nullOr bool;
+              default = null;
+              example = true;
+              description = "Whether to enable floating layout for krohnkite.";
+            };
+            monocle = {
+              enable = lib.mkOption {
+                type = with lib.types; nullOr bool;
+                default = null;
+                example = true;
+                description = "Whether to enable monocle layout for krohnkite.";
+              };
+              maximize = lib.mkOption {
+                type = with lib.types; nullOr bool;
+                default = null;
+                example = true;
+                description = "Fully maximize windows (no borders, no gaps) for monocle layout.";
```

Comment:
Isn't this the same as having all gaps set to `0`?

Reply:
No, maximize for monocle layout removes all borders (this includes window decorations) and gaps and makes the active window fullscreen